### PR TITLE
fix: issue where changing focus is causing unwanted scrolling behavior on iOS

### PR DESCRIPTION
**Pull request by alex-barstow**
## Description
Changing focus to a sub-menu (ex. by tapping the captions menu button) when a player is embedded in an iframe is resulting in unwanted scrolling behavior on iOS. @gkatsev is planning to refactor all focus handling soon, but this will serve as a temporary fix in the meantime. 

## Specific Additions
Modify the MenuButton `pressButton()` method to only change focus to a sub-menu on platforms other than iOS

### CODE REVIEW

File: src/js/menu/menu-button.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ import * as Dom from '../utils/dom.js';
 import * as Fn from '../utils/fn.js';
 import * as Events from '../utils/events.js';
 import toTitleCase from '../utils/to-title-case.js';
+import IS_IOS from '../utils/browser.js';
```
Comment (gkatsev) [inline src/js/menu/menu-button.js]: `IS_IOS` needs to be wrapped in curly braces when you want a named import
```js
import { IS_IOS } from '../utils/browser.js';
```

Comment (gkatsev) [reply src/js/menu/menu-button.js]: This is also why the build is failing.

Comment (alex-barstow) [reply src/js/menu/menu-button.js]: _facepalm_. Fixed.

**Comment by priyankapaladi** [comment]
I checked the fix on iOS 10.2.1 (iPad), using master branch of socrates and fix-menu-focus-ios branch of video.js. It works fine, I don't see it scroll abruptly anymore.
QA-LGTM

**Comment by gkatsev** [comment]
@alex-barstow so, since this only happens when the player is embedded in an iframe, maybe we should add that to the check? `if (ios && in-iframe) { dontFocus() }`. Thoughts?

**Comment by alex-barstow** [comment]
@gkatsev Changes made! I added the iframe check as a new utils function in dom.js

File: src/js/utils/dom.js [hunk 2]
```diff
@@ -86,6 +86,23 @@ export function isEl(value) {
 }
 
 /**
+ * Determines if the player is embedded in an iframe.
+ *
+ * @return {boolean}
+ *
+ */
+export function isPlayerInFrame() {
```
Comment (misteroneill) [inline src/js/utils/dom.js]: I propose changing this to `isInIframe` more generally. Might be pedantic or whatever, but that seems like a better name.

**Pull request merged**